### PR TITLE
Added proper readme

### LINKED ISSUE #5: Create a good readme that explains the project properly

### PR DESCRIPTION
**Pull request by yann510**
closes #5

### CODE REVIEW

File: renameProject.js [hunk 1]
```diff
@@ -33,6 +33,15 @@ async function main() {
     }
 
     await fs.promises.unlink("renameProject.js")
+    console.log("This is now YOUR project provided generously by:")
+    console.log(`
+███████ ████████  █████  ████████  ██████  ██████
```
Comment (GabrielBottari) [inline renameProject.js:38]: 👀

File: README.md [hunk 2]
```diff
@@ -1,2 +1,219 @@
...
+
+`npm i`
+
+Running the database:
+
+`npm run postgres`
+
+Running the api:
+
+`nx serve api`
```
Comment (GabrielBottari) [inline README.md:59]: maybe tell them to use npm start api instead?

Comment (GabrielBottari) [reply README.md:59]: since you're saying all commands are in package.json

**Review by GabrielBottari — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (yann510) [reply README.md:64]: Good point.

**Pull request merged**